Quadriga/pom.xml

```diff
@@ -360,6 +360,13 @@
 		</dependency>
 
 
+		<!-- JSoup -->
+		<dependency>
+			<groupId>org.jsoup</groupId>
+			<artifactId>jsoup</artifactId>
+			<version>1.8.3</version>
```

Comment:
Is there a reason you are not using the latest version of jsoup? If not, let's use the latest one.